File: TestNG Practise/src/Pracitse/browserInvoke.java
package Pracitse;

import org.openqa.selenium.WebDriver;
import org.openqa.selenium.chrome.ChromeDriver;
import org.testng.annotations.BeforeSuite;

public class browserInvoke {
	
	static WebDriver driver=null;
	
@BeforeSuite
public void ChromeB()
{
	System.setProperty("webdriver.chrome.driver", "C:\\Users\\Suresh Kumar\\Downloads\\chromedriver.exe");
	 driver = new ChromeDriver();
			
	System.out.println("test");
			
}

}

File: pythonProject/seleniumPython/browserInvoke.py
from selenium import webdriver


driver= webdriver.Chrome(executable_path="C:\\Users\\Suresh Kumar\\Downloads\\chromedriver_85\\chromedriver.exe")
#driver=webdriver.Firefox(executable_path="C:\\Users\\Suresh Kumar\\Downloads\\geckodriver-v0.25.0-win64\\geckodriver.exe")

driver.get("https://www.google.ca/")

print(driver.title)
driver.maximize_window()
driver.get("https://www.makemytrip.com/flights/")
driver.back()
print(driver.current_url)
driver.close()
